mobile/src/components/chat-conversation/components/message-form/message-form.tsx

```diff
@@ -47,6 +49,10 @@ const MessageForm: FC<Props> = ({ onSubmit }) => {
     }
   }, [messageChar]);
 
+  useEffect(() => {
+    setIsDisabledButton(!watchMessage);
```

Comment:
Let's remove validation schema
```suggestion
    setIsDisabledButton(!watchMessage.trim());
```